packages/core/scripts/liquidity-mining/CalculateBalancerLPRewards.js

```diff
@@ -20,29 +20,32 @@ const web3 = new Web3(new Web3.providers.HttpProvider(process.env.CUSTOM_NODE_UR
 const { toWei, toBN, fromWei } = web3.utils;
 
 const argv = require("minimist")(process.argv.slice(), {
-  string: ["poolAddress", "fromDate", "toDate"],
-  integer: ["week"],
-  boolean: ["test"]
+  string: ["poolAddress", "tokenName"],
+  integer: ["fromBlock", "toBlock", "week", "umaPerWeek", "blocksPerSnapshot"]
 });
 
-const UMA_PER_WEEK = toBN(toWei("25000"));
-const BLOCKS_PER_SNAPSHOT = 256;
```

Comment:
Why not make this a constant anymore? This shouldn't be affected by rolling right?

Reply:
Right but it's useful for testing. It defaults to 256 in the input params.